### ISSUE TITLE
Funky console messages with mobile theme editor

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1341]
@billreed63

Need to investigate some funky console messages.

```
Uncaught TypeError: Object $all has no method 'getParent' :8081/maqetta/:123
```

```
[theme editor style change command]  !FATAL! Rule not found in theme: .globalWidget .mblButton 
davinci/ve/themeEditor/commands/StyleChangeCommand:120
[theme editor style change command] !FATAL! Rule not found in theme: .globalWidget .mblButton
```

**Comment by JonFerraiolo** [comment]
The !FATAL! messages maybe can be addressed by removing the theme metadata entries that refer to globalWidget. At first glance, those theme metadata entries look unnecessary.

**Comment by JonFerraiolo** [comment]
Haven't yet found a way to reproduce the $all message above.

**Comment by JonFerraiolo** [comment]
Finally was able to reproduce the $all message and added a bandaid fix: 0b90104b3f
